SimTracker/SiPhase2Digitizer/python/phase2TrackerDigitizer_cfi.py

```diff
@@ -227,27 +234,32 @@
         AddNoisyPixels = False,
         AddInefficiency = False,
         AddThresholdSmearing = False,
+        UseReweighting = False,
```

Comment:
not sure to understand why you want to do that (in the case of the late re-weighting).
In any case please add a line of comment explaining why it is done after line 224 (in the proposed branch).

Reply:
This block is needed to feed a modifier. If that's the default behaviour, change it in the default configuration. If it's instead needed only for the premix stage1, please document it in the comments as already requested.

Reply:
Where is that supposed to be? I tried in`SiPixelSimBlock` part but that still tells me that the mixing is missing `UseReweighting`

Reply:
whic workflow did you use for testing?

Reply:
I used `20834.0`

Reply:
I am confused.
I tested this patch on top of your branch:
```diff
diff --git a/SimTracker/SiPhase2Digitizer/python/phase2TrackerDigitizer_cfi.py b/SimTracker/SiPhase2Digitizer/python/phase2TrackerDigitizer_cfi.py
index 47a0060e2c5..d9ec3dc3264 100644
--- a/SimTracker/SiPhase2Digitizer/python/phase2TrackerDigitizer_cfi.py
+++ b/SimTracker/SiPhase2Digitizer/python/phase2TrackerDigitizer_cfi.py
@@ -234,32 +234,32 @@ _premixStage1ModifyDict = dict(
         AddNoisyPixels = False,
         AddInefficiency = False,
         AddThresholdSmearing = False,
-        UseReweighting = False,
+        #UseReweighting = False,
     ),
     Pixel3DDigitizerAlgorithm = dict(
         AddNoisyPixels = False,
         AddInefficiency = False,
         AddThresholdSmearing = False,
-        UseReweighting = False,
+        #UseReweighting = False,
     ),
     PSPDigitizerAlgorithm = dict(
         AddNoisyPixels = False,
         AddInefficiency = False,
         AddThresholdSmearing = False,
-        UseReweighting = False,
+        #UseReweighting = False,
     ),
     PSSDigitizerAlgorithm = dict(
         AddNoisyPixels = False,
         AddInefficiency = False,
         AddThresholdSmearing = False,
         Phase2ReadoutMode = -1,
-        UseReweighting = False,
+        #UseReweighting = False,
     ),
     SSDigitizerAlgorithm = dict(
         AddNoisyPixels = False,
         AddInefficiency = False,
         AddThresholdSmearing = False,
-        UseReweighting = False,
+        #UseReweighting = False,
     ),
 )
```
and `runTheMatrix.py -l 20834.0 -t 4 -j 8 --ibeos` runs without problems for me.
Do I miss something?

Reply:
Yeah it indeed work, I guess this msg I referred to was at an earlier step in the development